userguide/content/en/_index.md

```diff
@@ -1,17 +1,15 @@
 ---
 title: Docsy
+description: A Hugo theme for creating great technical documentation sites
```

Comment:
Suggestion for the slogan, shuffling the words a bit: "A great Hugo theme for technical documentation sites"

Reply:
@geriom - thanks for the review. I reused the current site description. Let's discuss the possibility of updating it at our next PSC meeting (I've added an entry to our meeting notes).